src/utils/api.js

```diff
@@ -1,26 +1,47 @@
-import {shouldUseProxyUrl, proxyUrl, fhirUrl} from '../config';
+import {shouldUseProxyUrl, proxyUrl, fhirUrl, shouldUseCookie} from '../config';
 import {logErrors, replaceLocalhost} from '../utils/common';
 import store from '../store';
 
+const getHeaders = headers => {
+  let allHeaders = {
+    'Cache-Control': 'max-age=3600',
+    Accept: 'application/fhir+json;charset=utf-8',
+    ...headers,
+  };
+  const selectedServer = store.getState().app.selectedServer;
+  const requiresAuth = selectedServer.authType !== 'NO_AUTH';
+  if (requiresAuth && !shouldUseProxyUrl(selectedServer.url)) {
+    const token = store.getState().app.token;
+    allHeaders['Authorization'] = `Basic ${token}`;
+  }
+  return allHeaders;
+};
+
+const setCredentials = url => {
+  if (shouldUseCookie(url) && !url.includes(proxyUrl)) {
+    return 'include';
+  } else {
+    return 'omit';
```

Comment:
Do you think this should be `same-origin` since that's the default: https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials?